main.py

```diff
@@ -5,39 +5,33 @@
 
 
 def main() -> QuerySet:
-    Genre.objects.create(
-        name="Western"
-    ),
-    Genre.objects.create(
-        name="Action"
-    ),
-    Genre.objects.create(
-        name="Dramma"
-    ),
-    Actor.objects.create(
-        first_name="George",
-        last_name="Klooney"
-    ),
-    Actor.objects.create(
-        first_name="Kianu",
-        last_name="Reaves"
-    ),
-    Actress.objects.create(
-        first_name="Scarlett",
-        last_name="Keegan"
-    )
-    Actor.objects.create(
-        first_name="Will",
-        last_name="Smith"
-    ),
-    Actor.objects.create(
-        first_name="Jaden",
-        last_name="Smith"
-    ),
-    Actress.objects.create(
-        first_name="Scarlett",
-        last_name="Johansson"
-    ),
+    genres = ["Western", "Action", "Dramma"]
+    for genre in genres:
+        Genre.objects.create(
+            name=genre
+        ),
+
+    actors = [
+        ["George", "Klooney"],
+        ["Kianu", "Reaves"],
+        ["Will", "Smith"],
+        ["Jaden", "Smith"]
+    ]
+    for actor in actors:
```

Comment:
You can also unpack values right here for more readability, but it's fine as is.